packages/perspective-jupyterlab/src/ts/view.ts

```diff
@@ -12,11 +12,26 @@
 
 import { DOMWidgetView } from "@jupyter-widgets/base";
 import { PerspectiveJupyterWidget } from "./widget";
-import { PerspectiveJupyterClient } from "./client";
-
-import perspective from "@finos/perspective/dist/esm/perspective.js";
-
-function isEqual(a, b) {
+import { PerspectiveJupyterClient, PerspectiveJupyterMessage } from "./client";
+
+import perspective, {
+    PerspectiveWorker,
+    Table,
+    View,
+} from "@finos/perspective";
+
+import "@finos/perspective-viewer";
+import { proxy_table } from "@finos/perspective/api/table_api";
+
+// Type of first parameter to @finos/perspective UpdateCallback
+export type TableUpdate = {
+    port_id: number;
+    delta:
+        | Array<Record<string, Array<string | boolean | Date | number>>>
```

Comment:
I need to take another pass at these types, based on standup discussion they were right before

Reply:
Looks like `delta` for this type actually no longer supports `Array<...>` either, it is exclusively Arrow eg `ArrayBuffer`